gulpfile.js

```diff
@@ -270,6 +271,8 @@ const criticalCssConfig = (function () {
             article: global.concat(
                 /\.content-header__item_toggle--.*$/,
                 /\.contextual-data.*/,
+                /\.login-control.*/,
+                /\.speech-bubble.*/,
```

Comment:
Needed on any Hypothesis-enabled content (blog posts, interviews, Labs posts and press packages), all of which currently use `default`.

Reply:
`.contextual-data` too.